soh/soh/Enhancements/randomizer/randomizer_item_tracker.cpp

```diff
@@ -297,35 +297,44 @@ ItemTrackerNumbers GetItemCurrentAndMax(ItemTrackerItem item) {
             result.maxCapacity = result.currentCapacity = 100;
             result.currentAmmo = gSaveContext.inventory.gsTokens;
             break;
+        case ITEM_HEART_CONTAINER:
+            result.maxCapacity = result.currentCapacity = 8;
+            result.currentAmmo = gSaveContext.sohStats.heartContainers;
+            break;
+        case ITEM_HEART_PIECE:
+            result.maxCapacity = result.currentCapacity = 36;
+            result.currentAmmo = gSaveContext.sohStats.heartPieces;
+            break;
         case ITEM_KEY_SMALL:
             result.currentAmmo = MAX(gSaveContext.inventory.dungeonKeys[item.data], 0);
+            result.currentCapacity = gSaveContext.sohStats.dungeonKeys[item.data];
```

Comment:
i remember not liking the naming here because it's not actually capacity being tracked, might be nice to add a comment explaining the odd use (i think trying to rename could blow the scope pretty quick)

Reply:
Agreed to both 👍